src/components/ArcSlider/index.tsx

```diff
@@ -1,11 +1,25 @@
 import * as React from 'react';
 import styled from 'styled-components';
-import { constrainNumber, darken, getAngleBetweenPoints } from '../../utils';
+import { darken } from '../../utils';
 import { Box, BoxProps } from '../Box';
 import { Flex } from '../Flex';
 
 const ANGLERANGE = 135;
 
+type Point = { x: number; y: number };
+
+const getAngleBetweenPoints = (p1: Point, p2: Point) => {
+	return (Math.atan2(p2.y - p1.y, p2.x - p1.x) * 180) / Math.PI;
```

Comment:
Would be great to add a source for this calculation.

Reply:
@dimitrisnl I have no idea where it came from, but I am aiming to remove this component in the near future, so it doesn't really matter.